graphql/schema/types/task.graphql

```diff
@@ -86,6 +86,10 @@ type Task {
   taskFiles: TaskFiles!
   taskGroup: String
   taskGroupMaxHosts: Int
+  """
+  taskLogs returns the tail 100 lines of the task's logs.
+  """
+  taskLogs: TaskLogs!
```

Comment:
if you want, you can put this comment here (and the comments should probably be updated since we want to remove `query.taskLogs`)

<img width="551" alt="Screenshot 2023-02-23 at 9 33 48 AM" src="https://user-images.githubusercontent.com/47064971/220937190-f8c893a7-ad35-428e-884e-802651448dee.png">